packages/terra-collapse/tests/nightwatch/collapse-spec.js

```diff
@@ -0,0 +1,64 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+
+const screenshot = require('terra-toolkit').screenshot;
```

Comment:
I agree with Emily. What makes each test unique? Are there particular props or attributes the content or button should have? If so, we should test to make sure they are present on the dom for each test.